manifests/plugin/remote_execution/script.pp

```diff
@@ -70,4 +70,61 @@
       group             => $foreman_proxy::user,
     }
   }
+
+  if $mode == 'pull-mqtt' {
+    $mqtt_port = 1883
+    $mosquitto_config_dir = '/etc/mosquitto'
+    $mosquitto_ssl_dir = "${mosquitto_config_dir}/ssl"
+
+    class { 'mosquitto':
+      package_name => 'mosquitto',
+      config       => [
+        "listener ${mqtt_port}",
+        "acl_file ${mosquitto_config_dir}/foreman.acl",
+        "cafile ${mosquitto_ssl_dir}/ssl_ca.pem",
+        "certfile ${mosquitto_ssl_dir}/ssl_cert.pem",
+        "keyfile ${mosquitto_ssl_dir}/ssl_key.pem",
+        'require_certificate true',
+        'use_identity_as_username true',
+      ],
+      require      => File['/etc/mosquitto/foreman.acl'],
+    }
+
+    file { "${mosquitto_config_dir}/foreman.acl":
+      ensure  => 'file',
+      content => epp("${module_name}/plugin/foreman.acl.epp", { 'user' => $facts['networking']['fqdn'] }),
```

Comment:
One thing I can never remember, are our certs lower-case or not?

We have this explicit `lower_fqdn` variable:
https://github.com/theforeman/puppet-foreman_proxy/blob/bd47afb7d0b4d6042c361c2d9238543ea8f1c240/manifests/params.pp#L4